src/lib/core/WeaveTLVReader.cpp

```diff
@@ -1382,7 +1382,6 @@ WEAVE_ERROR TLVReader::VerifyElement()
                 return WEAVE_ERROR_INVALID_TLV_TAG;
             break;
         case kTLVType_Structure:
-        case kTLVType_Path:
```

Comment:
We need as case statement for kTLVType_Path that just breaks.  Otherwise the code will return WEAVE_ERROR_INCORRECT_STATE.

Reply:
fixed

Reply:
fixed